.github/pull_request_template.md

```diff
@@ -1,8 +1,5 @@
 Notes:
-* Pull requests cannot be accepted until:
--  The submitter has [accepted the online agreement here with a click through](https://developer.mbed.org/contributor_agreement/) 
-   or for companies or those that do not wish to create an mbed account, a slightly different agreement can be found [here](https://www.mbed.com/en/about-mbed/contributor-license-agreements/)
-- The PR follows the [mbed TLS coding standards](https://tls.mbed.org/kb/development/mbedtls-coding-standards)
+* Pull requests cannot be accepted until the PR follows the [contributing guidelines](../CONTRIBUTING.md)
```

Comment:
On the one hand, I think this template is too verbose. On the other hand, I think that the legal status of the contribution is important enough to warrant direct mention. Proposal:

> Pull requests cannot be accepted until the PR follows the [contributing guidelines](../CONTRIBUTING.md). In particular, you must indicate that you place your contribution under the project's license by including a `Signed-off-by` line in each commit message.

(the “including …” should point to a guide on how to do this once we have one.)

Reply:
OK, I'm not a fan of duplication in general but I'll repeat the Signed-off-by requirement.